setup.cfg

```diff
@@ -0,0 +1,21 @@
+[flake8]
+max-line-length = 120
+max-parameters-amount = 9
```

Comment:
Is it about max number of params per method? I think this is too much, Ideally, there should be 3, but I suggest  to set 5 and process extra param via **kwargs.